chain/chain/src/runtime/mod.rs

```diff
@@ -663,12 +663,7 @@ impl RuntimeAdapter for NightshadeRuntime {
                 current_protocol_version,
             ) {
                 Ok(_) => Ok(None),
-                Err(RuntimeError::InvalidTxError(err)) => {
```

Comment:
where is the crash mentioned in the PR description happening? is this part of the change because RuntimeErrors are leading to panics?

Reply:
I have exactly the same question 🙃 
In my understanding the only change here is to not return `Err` when `StorageError` happens, is that right?

Reply:
https://github.com/near/nearcore/blob/2e1df80fe4a343fefff98b45d4e6fe7eea0af8c2/chain/chain/src/runtime/mod.rs#L403-L416